backend/spherre/tests/test_models/test_notification_model.py

```
@@ -0,0 +1,27 @@
+        db.create_all()
+
+    def tearDown(self):
+        db.session.remove()
+        db.drop_all()
+        self.ctx.pop()
+
+    def test_create_notification_model():
+        # TODO: write test case for creating notifications
+        pass
+
+    def test_update_notification_model():
```

Pylint (3.3.7)

[error] Method 'test_update_notification_model' has no argument

(E0211)